### ISSUE TITLE
My student pack has already expired.

### ISSUE THREAD

**Issue by Laeng** [education/GitHubGraduation-2022#3001]
In Korea, the semester starts in March and ends in February.
My student pack already expired in March, what should I do?

#2987

**Comment by Laeng** [comment]
> Same situation here. I am also a South Korean, I graduated in Feb. 2022, but my GitHub Education Student Pack is currently expired. I will link my pull request here. #3010

Hi htcrefactor! 

I just applied with my graduation certificate attached to the student discount request.
How about you applying too?

(The application number has a unique number in the address of the page indicating that the application has been completed.)

**Comment by Laeng** [comment]
> Same situation here. I am also a South Korean, I graduated in Feb. 2022, but my GitHub Education Student Pack is currently expired. I will link my pull request here. #3010

Hey, htcrefactor!
I have a good news.
My GitHub student develop pack application has been approved and my pull request has passed the validation test!

**Comment by htcrefactor** [comment]
Okay, let me give it a try. 감사합니다! :)

**Comment by Laeng** [comment]
> Student Developer Packs are eligible to students who are currently maintaining their student status. Since we graduated, I think we fail to meet this condition. Is renewing the Student Developer Pack sort of a workaround hack? If so, I think receiving direct feedback from maintainers would be better.

Yes, Your right, the Student Developer Pack is only available to students.
I hope your problem is well resolved.

**Comment by htcrefactor** [comment]
Thanks, I'll try to get a sound solution with the information you provided. 감사합니다!

**Comment by htcrefactor** [comment]
@elisemoe Hi, I would love to get feedback on this issue. I graduated at Feb 2022, and currently SDP has been expired.
I think I'm in the same situation as this [post](https://github.com/orgs/github-community/discussions/99) in GitHub Discussions.

This is [my post](https://github.com/orgs/github-community/discussions/108) in GitHub Discussions, and [this](https://github.com/education/GitHubGraduation-2022/pull/3010) is my pull request which got blocked.

**Comment by goduguaparna** [comment]
my student developer pack is not approving ....what to upload in proof other than college id.iam unable to understand what to upload ..plz respond for the issue plz...........

**Comment by ullaskunder3** [comment]
@goduguaparna 
- If the dates or schedule mentioned in your uploaded image do not match the eligibility criteria, might be the problem.
- If the image you uploaded doesn't clearly identify your current academic status or if the uploaded image is blurry
